src/catkin_pkg/package_templates.py

```diff
@@ -41,6 +41,8 @@
 from catkin_pkg.package import Package
 from catkin_pkg.package import PACKAGE_MANIFEST_FILENAME
 from catkin_pkg.package import Person
+from catkin_pkg.cmake import get_metapackage_cmake_template_path
+from catkin_pkg.cmake import configure_file
```

Comment:
Sure, no problem, I would do the same. 
I didn't know PEP8, thought it was a ROS style guide or something internal, but I see now. As you surely noticed, I'm not a pythoner. In cmake.py they are not defined in order though.